pkg/controller/plan/kubevirt.go

```diff
@@ -983,6 +983,18 @@ func (r *KubeVirt) findTemplate(vm *plan.VMStatus) (tmpl *template.Template, err
 
 func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume, configMap *core.ConfigMap, pvcs *[]core.PersistentVolumeClaim, v2vSecret *core.Secret) (pod *core.Pod, err error) {
 	volumes, volumeMounts, volumeDevices := r.podVolumeMounts(vmVolumes, configMap, pvcs)
+	resourceReq := core.ResourceRequirements{}
+
+	// Request access to /dev/kvm via Kubevirt's Device Manager
+	// That is to ensure the appliance virt-v2v uses would not
+	// run in emulation mode, which is significantly slower
+	if Settings.VirtV2vRequireKVM {
+		resourceReq = core.ResourceRequirements{
+			Limits: core.ResourceList{
+				"devices.kubevirt.io/kvm": resource.MustParse("1"),
+			},
```

Comment:
why not to just set the Limits of resourceReq here then (but to override the entire struct)?

Reply:
agreed, done